### DISCUSSION TITLE
How can i move a project to a different organization?

### DISCUSSION THREAD

**Discussion by s0er3n** [supabase/supabase#6445] [Questions] [answered]
I have 2 projects in 1 organization but i want to move 1 project into another organization.

**Comment by GaryAustin1** [comment, marked as answer]
You might ask support, but other than that you would have to manually migrate it: https://supabase.com/docs/guides/database#migrating-between-projects

**Reply by devidw** [reply]
New docs link is: https://supabase.com/docs/guides/database/migrating-between-projects

**Comment by pombredanne** [comment]
You will need this script for "new style" projects to migrate also issues. https://gist.github.com/keshav-space/f0afa93cb6ec1b9a39299deef9d6858b
